docs/pages/docs/config/config.md

```diff
@@ -32,6 +32,19 @@ We will cover each of these options below.
 The configuration object has a TypeScript type of `KeystoneConfig`, which can be imported from `@keystone-6/core/types`.
 This type definition should be considered the source of truth for the available configuration options.
 
+Note: It is important to pass a `TypeInfo` type argument to the config function as it ensures proper typing for the [Keystone Context](../context/overview.md). This type is automatically created in the `node_modules/.keystone/types` directory. You can customize the output path of the generated type by specifying it in the config object.
+
+```typescript
+import { TypeInfo } from ".keystone/types";
+
+export default config<TypeInfo>({
+  //other config options,
+  types: {
+    path: ".keystone/types"
+  }
```

Comment:
```suggestion
  // ...
```

This is quite an advanced option, and I don't think we want to introduce that on this page

Reply:
Alright...
I would then also change the import I used in this example to default
```ts
import { TypeInfo } from "node_modules/.keystone/types";
```

Although, I think it would be better if the default path is `.keystone/types.ts`

Reply:
The default path is `.keystone/types`

Reply:
oh, okay then...